ouroboros-consensus/src/Ouroboros/Consensus/Util/IOLike.hs

```diff
@@ -34,6 +34,8 @@ module Ouroboros.Consensus.Util.IOLike (
   , diffTime
     -- *** MonadDelay
   , MonadDelay(..)
```

Comment:
```suggestion
  , MonadDelay
```
Idea: just export `MonadDelay` but not its sole method `threadDelay`). This will prevent us from accidentally using `threadDelay` in the future.

It's still not perfect, as in some places we might import the following directly:
```haskell
import           Control.Monad.Class.MonadTimer (MonadDelay (..))
```